pkgs/development/compilers/dotnet/update.sh

```diff
@@ -89,6 +89,43 @@ sdk_packages () {
       "Microsoft.AspNetCore.App.Runtime.win-arm64" \
       "Microsoft.AspNetCore.App.Runtime.win-x64" \
       "Microsoft.AspNetCore.App.Runtime.win-x86" \
+    )
+
+    local nuget_url="$(curl -f "https://api.nuget.org/v3/index.json" | jq --raw-output '.resources[] | select(."@type" == "PackageBaseAddress/3.0.0")."@id"')"
+
+    for pkg in "${pkgs[@]}"; do
+        local url hash
+        url="${nuget_url}${pkg,,}/${version,,}/${pkg,,}.${version,,}.nupkg"
+        hash="$(nix-prefetch-url "$url")"
+        echo "      (fetchNuGet { pname = \"${pkg}\"; version = \"${version}\"; sha256 = \"${hash}\"; })"
+    done
```

Comment:
I think this logic should get extracted to its own function to avoid code repitition and mirror potential changes automatically.

Reply:
Fixed